mediapipe/graphs/instantmotiontracking/calculators/matrices_manager_calculator.cc

```diff
@@ -114,13 +118,32 @@ class MatricesManagerCalculator : public CalculatorBase {
     // This returns a scale factor by which to alter the projection matrix for
     // the specified render id in order to ensure all objects render at a similar
     // size in the view screen upon initial placement
-    const float GetDefaultRenderScale(const int render_id) {
+    const DiagonalMatrix3f GetDefaultRenderScaleDiagonal(const int render_id, const float user_scale_factor, const float gif_aspect_ratio) {
+      float combined_scale = 1.0f;
```

Comment:
nit: can condense a little by having combined_scale start off as user_scale_factor, and then can just *= 160.0f or *= 5.0f below.